README.md

```diff
@@ -44,12 +44,27 @@ Enter your DSN in the NuxtJS config file. Additional config settings can be foun
 
 In a Vue component, `Sentry` is available as `this.$sentry`, so we can call functions like
 
-```
+``` js
 this.$sentry.captureException(new Error('example'))
 ```
 
 where `this` is a Vue instance.
 
+### Usage Server-Side
```

Comment:
The description you wrote applies to client-side too. I think instead of using 'server-side' term, you should switch to `Usage inside asyncData` without specific if it's for server or client.

Reply:
Yes! For full disclosure, I had the issue inside of the `nuxtServerInit` store function and after thinking bout it, it seemed more accessible to present the usage in the more common `asyncData` function. 
The idea here is to show the fact that the $sentry key is injected in the `app` object of the global context. Regarding this, I'm wondering what is more understandable from a beginner point of view ?

Reply:
Having `asyncData`-specific information is useful, I think. More generic description could say that `$sentry` is injected into `context.app`, Vue instance (accessible from script and template as `this.$sentry`) and Store (don't know details of that though).

Reply:
Allright, I'll ammend the PR this evening with this elements in head !